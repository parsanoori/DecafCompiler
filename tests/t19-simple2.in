define PI 3.14159265359
define FOR100 for ( i = 0; i < 100; i++)
int main () {
    int i;
    FOR100 {
    Print ( i , PI ) ;
    }
}
