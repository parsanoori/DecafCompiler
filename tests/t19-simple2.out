int
T_ID main
(
)
{
int
T_ID i
;
for
(
T_ID i
=
T_INTLITERAL 0
;
T_ID i
<
T_INTLITERAL 100
;
T_ID i
++
)
{
Print
(
T_ID i
,
T_DOUBLELITERAL 3.14159265359
)
;
}
}
